src/assets/images/check.svg

```diff
@@ -1,3 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 13l4 4L19 7" />
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="4" d="M5 13l4 4L19 7" />
```

Comment:
<img width="209" alt="Screenshot 2021-09-16 at 9 45 54 AM" src="https://user-images.githubusercontent.com/13185699/133548489-4c2e10ac-5f6e-468a-beb7-184449be9b07.png">
why not use`check-solid`? It looks almost exactly the same.

Reply:
This is how it looks like with check.svg (with higher stroke width)
<img width="482" alt="Screenshot 2021-09-20 at 10 17 11 AM" src="https://user-images.githubusercontent.com/20169829/133958081-125c4fdc-2352-4205-9a96-4cd264c403f5.png">

This is how it looks like with check-solid.svg
![image](https://user-images.githubusercontent.com/20169829/133958042-cb2d3836-ee72-4840-914e-e1962508f76c.png)

Going with the `check.svg` here as the same icon sizes are applied to all the icons on the scorecard, and in that case, `check-solid.svg` looks too big, and I don't want to configure different sizes for every different icon

Reply:
let's replace check-solid with check everywhere else then and delete check-solid.